PyRDF/CallableGenerator.py

```diff
@@ -2,18 +2,32 @@
 
 class CallableGenerator(object):
     """
-    Class that generates
-    mapper and reducer functions
+    Class that generates a callable
+    to parse a PyRDF graph.
+
+    Attributes
+    ----------
+    root_node
+        Head node a PyRDF graph.
```

Comment:
of a PyRDF graph?

Reply:
@etejedor , I meant Python RDataFrame graph. Since this isn't the CPP RDF graph, I made it Python explicitly

Reply:
Shall then change this to " Python RDataFrame graph" then @etejedor ?

Reply:
I think he means you missed **of**  right before **a PyRDF**

Reply:
Oh yes. I'm sorry that I didn't notice it. Thank @JavierCVilla 😬